src/gt4py/next/program_processors/runners/dace_fieldview/utility.py

```diff
@@ -126,3 +130,12 @@ def get_map_variable(dim: gtx_common.Dimension) -> str:
     """
     suffix = "dim" if dim.kind == gtx_common.DimensionKind.LOCAL else ""
     return f"i_{dim.value}_gtx_{dim.kind}{suffix}"
+
+
+def get_neighbors_field_type(offset: str, dtype: dace.typeclass) -> ts.FieldType:
+    """Utility function to obtain the descriptor for a local field of neighbors."""
+    scalar_type = as_scalar_type(str(dtype.as_numpy_dtype()))
+    return ts.FieldType(
```

Comment:
While integrating the typing system I realized that this type is incorrect. I should use `dtype=ListType` instead of adding a local dimension:
```
FieldType(
  dims=[
    Dimension(value='Vertex', kind=<DimensionKind.HORIZONTAL: 'horizontal'>)
  ],
  dtype=ListType(element_type=ScalarType(kind=<ScalarKind.FLOAT64: 1064>, shape=None))
)
```